source/administration/config-in-database.rst

```diff
@@ -34,7 +34,7 @@ Example output:
 
 .. code-block:: text
 
-   SqlSettings.DataSource: "mmuser:really_secure_password@tcp(127.0.0.1:3306)/mattermost?charset=utf8mb4,utf8\u0026readTimeout=30s\u0026writeTimeout=30s"
+   SqlSettings.DataSource: "mmuser:really_secure_password@tcp(127.0.0.1:3306)/mattermost?charset=utf8mb4,utf8\u0026QueryTimeout=30s\u0026writeTimeout=30s"
```

Comment:
@cwarnermm 

We can just remove the whole `readTimeout` parameter from these references without any replacement.
The `QueryTimeout` I was referring too is in the configuration, under the `SqlSettings` section.

Reply:
Thanks so much, @streamer45! I'll remove the ``QueryTimeout`` references that have been updated as part of this PR. The only config setting reference to ``readTimeout`` I've found is here: https://docs.mattermost.com/administration/config-settings.html#read-timeout 

Is this the setting you're referring to that you've updated?

Reply:
The references you identified are correct but instead of replacing them we just need to remove them entirely, so for example this would become:

```suggestion
   SqlSettings.DataSource: "mmuser:really_secure_password@tcp(127.0.0.1:3306)/mattermost?charset=utf8mb4,utf8\u0026writeTimeout=30s"
```

the `readTimeout` parameter that we ignore is only part of the database DSN (connection URL). Instead now the app relies on the already existing `SqlSettings.QueryTimeout` which is unchanged.

Reply:
Perfect! Thank you so much for clarifying what's needed, @streamer45!